test/parsing/errors/useImportNeedsModule-detailed.good

```diff
@@ -1,5 +1,5 @@
 ─── syntax in useImportNeedsModule.chpl:1 [UseImportNeedsModule] ───
-  'use' statements must refer to module or 'enum' symbols.
+  'use' statements must refer to module or enum symbols.
```

Comment:
I expected to see a new end-to-end test here that would exercise the new error message(s) for improper matches. Is it a super big PITA or something?

Reply:
I added tests for imports and uses.